app/clients/hedera/hedera.go

```diff
@@ -1,3 +1,58 @@
 package hedera
 
-// TODO hedera client for interacting with hedera nodes
+import (
+	"encoding/json"
+	"fmt"
+	hederasdk "github.com/hashgraph/hedera-sdk-go"
+	"github.com/limechain/hedera-eth-bridge-validator/app/process/model/transaction"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type Client struct {
+	mirrorAPIAddress string
+	mirrorClient     *hederasdk.MirrorClient
+	httpClient       *http.Client
+}
+
+func (hedera Client) GetMirror() *hederasdk.MirrorClient {
+	return hedera.mirrorClient
+}
+
+func NewClient(mirrorAPIAddress, mirrorNodeClientAddress string) *Client {
```

Comment:
In the two parameters, you use once just `mirror` and once `mirrorNode`. I'd preffert to stick to `mirrorNode` in both occasions.